nuget/Microsoft.NETCore.Runtime.Mono.LLVM.Tools/runtime.Linux.Microsoft.NETCore.Runtime.Mono.LLVM.Tools.props

```diff
@@ -7,5 +7,7 @@
     <File Include="$(_LLVMInstallDir)\bin\llvm-as.dbg" TargetPath="tools\$(PackageTargetRuntime)\bin\" />
     <File Include="$(_LLVMInstallDir)\bin\opt" TargetPath="tools\$(PackageTargetRuntime)\bin\" />
     <File Include="$(_LLVMInstallDir)\bin\opt.dbg" TargetPath="tools\$(PackageTargetRuntime)\bin\" />
+    <File Include="$(_LLVMInstallDir)\bin\FileCheck" TargetPath="tools\$(PackageTargetRuntime)\bin\" />
+    <File Include="$(_LLVMInstallDir)\bin\FileCheck.dbg" TargetPath="tools\$(PackageTargetRuntime)\bin\" />
```

Comment:
so, can we add here `llvm-mca` as well or it's too heavy?

Reply:
What is `llvm-mca` ? Just so we have a description written here.

I can check the dependencies to see what it needs.

Reply:
A handy tool to e.g. redirect DOTNET_JitDisasm to and get assembly code efficiency analysis, similar to perfscore but more closer to reality.

Reply:
Example: https://godbolt.org/z/9faz76sPY

Reply:
Looks like we should be able to include `llvm-mca` - it's standalone.

Reply:
> A handy tool to e.g. redirect DOTNET_JitDisasm to and get assembly code efficiency analysis, similar to perfscore but more closer to reality.

Wow, that would be very handy.